vision_layer/abc/CMakeLists.txt

```diff
@@ -0,0 +1,199 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(beginner_tutorials)
```

Comment:
Package name is not same as `abc`. Please follow a fixed pattern, and directory structure correctly.